provision/aws/route53/route53_create_failover_lambda.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+set -e -o pipefail
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/route53_common.sh
+
+if [ -z "${HEALTH_CHECK_ID}" ]; then
+  echo "HEALTH_CHECK_ID variable must be set"
+  exit 1
+fi
+
+ALARM_NAME=${HEALTH_CHECK_ID}
+TOPIC_NAME=${HEALTH_CHECK_ID}
+FUNCTION_NAME=${HEALTH_CHECK_ID}
+ROUTE53_REGION="us-east-1"
```

Comment:
@ryanemerson isn't Route53 a global resource, just for my understanding should we be still providing a route53 region for it ?

Reply:
It's a global resource, but it seems CloudWatch alarms associated with Route53 must be created in `us-east-1`. It should be possible to deploy the Lambda function to a different region, as we use an SNS topic, but given that the Lambda only updates Route53 I figured it was easier to just keep it as one region.

https://stackoverflow.com/questions/44787306/can-a-cloudwatch-event-in-one-region-trigger-a-lambda-in-another-region-of-aws

Reply:
Thank you, I was not aware of that.

Reply:
i'll add this to the script and merge it